packages/module/src/LogSnippet/LogSnippet.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { CodeBlock, CodeBlockCode, Flex, FlexItem, Text, TextVariants } from '@patternfly/react-core';
+
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  logSnippet: {
+    borderLeft: '2px solid var(--pf-v5-global--danger-color--100)',
```

Comment:
also, we could use the PF variable for the border width - `--pf-v5-global--BorderWidth--md`